ShareExtension/CustomShareExtension/CustomShareViewController.swift

```diff
@@ -10,27 +10,40 @@ import SwiftUI
 import MobileCoreServices
 
 class CustomShareViewController: UIViewController {
+
```

Comment:
Remove this blank line. Try to follow the same coding rule in all of the project. Find the more appropriate coding style that suits you and follow it. I.e. https://google.github.io/swift/#file-comments